plugins/source/aws/resources/services/organizations/organizational_unit_accounts.go

```diff
@@ -0,0 +1,48 @@
+package organizations
+
+import (
+	"context"
+
+	"github.com/apache/arrow/go/v13/arrow"
+	"github.com/aws/aws-sdk-go-v2/service/organizations"
+	"github.com/aws/aws-sdk-go-v2/service/organizations/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v3/schema"
+	"github.com/cloudquery/plugin-sdk/v3/transformers"
+)
+
+func organizationalUnitAccounts() *schema.Table {
+	tableName := "aws_organizations_organizational_unit_accounts"
+	return &schema.Table{
+		Name:        tableName,
+		Description: `https://docs.aws.amazon.com/organizations/latest/APIReference/API_ListAccountsForParent.html`,
+		Resolver:    fetchAccountsForParent,
+		Transform:   transformers.TransformWithStruct(&types.Account{}, transformers.WithPrimaryKeys("Arn")),
+		Columns: []schema.Column{
+			client.DefaultAccountIDColumn(true),
```

Comment:
Ideally this should be `request_account_id` but that would require changing the parent table in a backwards incompatible way